app/views/section1/trader_reference.scala.html

```diff
@@ -82,8 +81,6 @@
             content = HtmlContent(expanderContent)
         ))
 
-        @tariffExpander(TraderReference, request.declarationType)(messages, appConfig)
```

Comment:
The injection of `AppConfig` at L38, as well as the `@import`, should be removed